### ISSUE TITLE
NVidia 435.21 failing to build with latest 5.4 kernel

### ISSUE THREAD

**Issue by mdedetrich** [NixOS/nixpkgs#74747]
**Describe the bug**
Nvidia propriety module failing to build with latest kernel

```
Building linux driver against kernel: /nix/store/lbl993a8wdv9q2zai1jkcaqscwgjc47y-linux-5.4-dev
make[1]: Entering directory '/nix/store/lbl993a8wdv9q2zai1jkcaqscwgjc47y-linux-5.4-dev/lib/modules/5.4.0/source'
make[2]: Entering directory '/nix/store/lbl993a8wdv9q2zai1jkcaqscwgjc47y-linux-5.4-dev/lib/modules/5.4.0/build'
The Module.symvers file is missing, or does not contain any
symbols exported from the kernel. This could cause the NVIDIA
kernel modules to be built against a configuration that does
not accurately reflect the actual target kernel.
The Module.symvers file check can be disabled by setting the
environment variable IGNORE_MISSING_MODULE_SYMVERS to 1.
make[3]: *** [/build/kernel/Kbuild:181: module_symvers_sanity_check] Error 1
make[3]: *** Waiting for unfinished jobs....
make[2]: *** [/nix/store/lbl993a8wdv9q2zai1jkcaqscwgjc47y-linux-5.4-dev/lib/modules/5.4.0/source/Makefile:1652: /build/kernel] Error 2
make[2]: Leaving directory '/nix/store/lbl993a8wdv9q2zai1jkcaqscwgjc47y-linux-5.4-dev/lib/modules/5.4.0/build'
make[1]: *** [Makefile:179: sub-make] Error 2
make[1]: Leaving directory '/nix/store/lbl993a8wdv9q2zai1jkcaqscwgjc47y-linux-5.4-dev/lib/modules/5.4.0/source'
make: *** [Makefile:81: modules] Error 2
builder for '/nix/store/303yil6ivmhrrjklsirblcppa58wxidw-nvidia-x11-435.21-5.4.drv' failed with exit code 2
cannot build derivation '/nix/store/ib0nb2pypg14pfry85acf9ngh8grvlp1-etc.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/5l2a9vpmynn373yrg1b6hia38q97wchv-kernel-modules.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/llvds6yv6anaww9d18mk938kjk71kvki-linux-5.4-modules.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/c66imj4a0w8z1sdfrr81gylj6mm26k6f-system-path.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/8h9q8yj2r3cczzqr3c5w26amxn3ww6q5-nixos-system-mdedetrich-nixos-laptop-20.03pre203434.3ccbc8d8915.drv': 1 dependencies couldn't be built
error: build of '/nix/store/8h9q8yj2r3cczzqr3c5w26amxn3ww6q5-nixos-system-mdedetrich-nixos-laptop-20.03pre203434.3ccbc8d8915.drv' failed
```

Full stacktrace can be seen at https://gist.github.com/mdedetrich/7413493f7a32e8af802ce957737079c8

**Comment by veprbl** [comment]
Duplicate of #74523

**Comment by veprbl** [comment]
Should be fixed by #73167
